src/js/angular/core/services/repositories.service.js

```diff
@@ -57,13 +52,17 @@ repositories.service('$repositories', ['$http', 'toastr', '$rootScope', '$timeou
         };
 
         this.resetActiveRepository = function () {
-            let repositoriesFromLocation = that.repositories.get(this.repository.location);
+            const repository = {
+                id: LocalStorageAdapter.get(LSKeys.REPOSITORY_ID) || '',
+                location: LocalStorageAdapter.get(LSKeys.REPOSITORY_LOCATION) || ''
+            };
```

Comment:
The repository creation object from local store can be extracted as a function and can be used everywhere